packages/rum/src/resourceUtils.ts

```diff
@@ -39,29 +39,49 @@ export function computeResourceKind(timing: PerformanceResourceTiming) {
   return ResourceKind.OTHER
 }
 
+function formatTiming(start: number, end: number) {
+  if (start <= 0) {
+    return undefined
+  }
+  return { duration: msToNs(Math.max(end - start, 0)), start: msToNs(start) }
```

Comment:
I am not so sure about doing this fix right now.

I would be in favor collecting info to understand better the issue and then discuss with the product what is the best way to handle it. We could prefer to not send the timing instead of setting the duration to 0.

Reply:
Ok, I opened #251 for this.